vagrant/plugins/vagrant-hypconfigmgmt/lib/vagrant-hypconfigmgmt/command.rb

```diff
@@ -269,15 +269,20 @@ def inform_if_gatling_not_installed(env)
 
     def ensure_vagrant_box_type_configured(env)
       settings = retrieve_settings()
-      case settings['php']['version']
-        when 5.5
-          env[:ui].info("Will use PHP 5.5. If you want PHP 7 instead change the php version in local.yml.")
-          settings['vagrant']['box'] = 'hypernode_php5'
-          settings['vagrant']['box_url'] = 'http://vagrant.hypernode.com/customer/php5/catalog.json'
-        when 7.0
-          env[:ui].info("Will use PHP 7. If you want PHP 5.5 instead change the php version in local.yml.")
-          settings['vagrant']['box'] = 'hypernode_php7'
-          settings['vagrant']['box_url'] = 'http://vagrant.hypernode.com/customer/php7/catalog.json'
+      if !settings['ubuntu_version'].nil? and settings['ubuntu_version'] == 'xenial'
```

Comment:
Oh right didn't realize that you can do this in Ruby. In Python you'd get a KeyError
```
irb(main):008:0> a = Hash.new()
=> {}
irb(main):009:0> a['doesnotexist'] == 'xenial'
=> false
```

```
>>> a = dict()
>>> a['doesnotexist'] == 'xenial'
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
KeyError: 'doesnotexist'
```